### PR TITLE
Updates for v4 installation steps

### LINKED ISSUE #124: Documentation update
Bring docs up to date for the current mainnet (some of this already done in open PRs, will verify) to ensure easier onboarding for validators.

### PR DESCRIPTION
**Pull request by cbrit**
Closes #124

### CODE REVIEW

**Review by JimR8 — CHANGES_REQUESTED** [review]
New docs are looking good! I have not rebuilt a node using them but took a quick look over the commands and with the addition of the services files you have added the only other comments i could add is propose a change below to replace wildcard with binary name incase 'install' directory exists and contains other files 

Original command: wget https://github.com/PeggyJV/steward/releases/download/v2.0.2/steward && chmod +x * && sudo mv * /usr/bin
Proposed command: wget https://github.com/PeggyJV/steward/releases/download/v2.0.2/steward && chmod +x steward && sudo mv steward /usr/bin

Also suggestion for your consideration that creating service account users as part of the process could be a good security improvement as well as allow the docs to contain correct paths when referencing users home directories for keystore and configs currently the process uses a user called ubuntu. This would require a lot of changes to the docs and files and if desirable maybe better added at a later stage so we can ship the changes and get the docs updated.

File: README.md [hunk 1]
```diff
@@ -25,92 +25,98 @@ The initial release of the Sommelier blockchain will consist of a standard cosmo
...
+# Install Steward
+wget https://github.com/PeggyJV/steward/releases/download/v2.0.2/steward && chmod +x * && sudo mv * /usr/bin
 
 # Install Geth
-wget https://gethstore.blob.core.windows.net/builds/geth-linux-amd64-1.10.4-aa637fd3.tar.gz && tar -xvf geth-linux-amd64-1.10.4-aa637fd3.tar.gz && sudo mv geth-linux-amd64-1.10.4-aa637fd3/geth /usr/bin/geth && rm -rf geth-linux-amd64-1.10.4-aa637fd3*
+wget https://gethstore.blob.core.windows.net/builds/geth-linux-amd64-1.10.23-d901d853.tar.gz && tar -xvf geth-linux-amd64-1.10.23-d901d853.tar.gz && sudo mv geth-linux-amd64-1.10.23-d901d853/geth /usr/bin/geth && rm -rf geth-linux-amd64-1.10.23-d901d853*
 
 # Install Sommelier
-wget https://github.com/PeggyJV/sommelier/releases/download/v3.1.1/sommelier_3.1.1_linux_amd64.tar.gz && tar -xf sommelier_3.1.1_linux_amd64.tar.gz && sudo mv sommelier /usr/bin && rm -rf sommelier_3.1.1_linux_amd64* LICENSE README.md
+wget https://github.com/PeggyJV/sommelier/releases/download/v4.0.1/sommelier_4.0.1_linux_amd64.tar.gz && tar -xf sommelier_4.0.1_linux_amd64.tar.gz && sudo mv sommelier /usr/bin && rm -rf sommelier_4.0.1_linux_amd64* LICENSE README.md
```
Comment (EricBolten) [inline README.md:61]: It's nice that we provide direct commands, but every time the version of the node bumps (e.g. we just had a patch update for Dragonberry) we'd have to update these commands. Is there a way to point to "latest" or something?

File: README.md [hunk 2]
```diff
@@ -25,92 +25,98 @@ The initial release of the Sommelier blockchain will consist of a standard cosmo
...
-# - a key named "signer" with funds on connected ETH chain in the gorc keystore
+# - a key named "orchestrator" with funds on the cosmos chain in the steward keystore
+# - a key named "signer" with funds on connected ETH chain in the steward keystore
 # - a key named "validator" with funds on the cosmos chain in the sommelier keystore
 
 # Add the peers from contrib/mainnet/sommelier-3/peers.txt to the ~/.sommelier/config/config.toml file
 nano ~/.sommelier/config/config.toml
 
-# pull the genesis file 
+# pull the genesis file
```
Comment (EricBolten) [inline README.md:113]: Given there has been an upgrade in the intervening time, I don't think downloading the v4 binary will work for syncing from the genesis, which started on v3. Should we provide some instructions to use Cosmovisor or how to handle this manually?

File: README.md [hunk 3]
```diff
@@ -120,24 +126,39 @@ sommelier tx staking create-validator \
...
-    $(gorc --config $HOME/gorc/config.toml keys cosmos show orchestrator) \ # orchestrator address (this must be run manually and address extracted)
-    $(gorc --config $HOME/gorc/config.toml keys eth show signer) \ # eth signer address
-    $(gorc --config $HOME/gorc/config.toml sign-delegate-keys signer $(sommelier keys show validator --bech val -a)) \ 
-    --chain-id sommelier-3 \ 
-    --from validator \ 
+    $(steward --config $HOME/steward/config.toml keys cosmos show orchestrator) \ # orchestrator address (this must be run manually and address extracted)
+    $(steward --config $HOME/steward/config.toml keys eth show signer) \ # eth signer address
+    $(steward --config $HOME/steward/config.toml sign-delegate-keys signer $(sommelier keys show validator --bech val -a)) \
+    --chain-id sommelier-3 \
+    --from validator \
```
Comment (EricBolten) [inline README.md:140]: IIRC some folks were having issues with setting gas correctly for `set-delegate-keys`, do we need to specify it manually?

File: README.md [hunk 4]
```diff
@@ -161,7 +182,7 @@ gorc eth-to-cosmos \
     --erc20-address="0x0000000000000000000000000000000000000000" \
     --amount="1.3530000" \
     --cosmos-destination="$(sommelier keys show orchestrator -a)"
-    
+
```
Comment (EricBolten) [inline README.md:185]: We may want to remove these from the default README, there's really no need for an individual validator to bridge assets during their setup, and the extra traffic spends ETH.

File: contrib/mainnet/sommelier-3/config.toml [hunk 5]
```diff
@@ -22,4 +22,14 @@ amount = 0.000
 denom = "usomm"
 
 [metrics]
-listen_addr = "127.0.0.1:3000"
\ No newline at end of file
+listen_addr = "127.0.0.1:3000"
+
+[keys]
+delegate_key = "orchestrator" # Edit if your orchestrator's Cosmos key name is different
```
Comment (EricBolten) [inline contrib/mainnet/sommelier-3/config.toml:28]: Should we add a comment indicating `delegate_key` is only used by steward and not the orchestrator?

**Review by EricBolten — COMMENTED** [review]
A couple thoughts.

File: README.md (hunk 1 above)
Comment (cbrit) [reply README.md:61]: There is a `latest` page but I don't see how you could select a specific binary architecture without being explicit about the file name. Maybe we just make this a manual step?

**Pull request closed without merging**